modules/internal/ChapelBase.chpl

```diff
@@ -1539,7 +1529,7 @@ module ChapelBase {
   pragma "no doc"
   inline proc _defaultOf(type t) where t: _sync_aux_t return _nullSyncVarAuxFields;
   pragma "no doc"
-  inline proc _defaultOf(type t) where t == _task_list return _nullTaskList;
+  inline proc _defaultOf(type t) where t == c_void_ptr return _nullVoidPtr;
```

Comment:
Isn't the c_void_ptr _defaultOf already defined at https://github.com/chapel-lang/chapel/blob/master/modules/internal/CPtr.chpl#L94-L96